student.library/src/main/java/tds/student/sql/data/OpportunityInstance.java

```diff
@@ -37,9 +39,25 @@ public UUID getBrowserKey () {
     return _browserKey;
   }
 
+  @JsonProperty("ExamId")
```

Comment:
I am following the existing style.  The current code base uses Title Casing so I carried it forward in this class.